Automattic/course
# Remove Landing Page pattern

The Landing Page pattern will be part of the Sensei LMS plugin, not the theme.

## Testing Instructions
- Create a new page.
- Open the block inserter and click on _Patterns_.
- Select _Sensei LMS_ from the dropdown.
- Check that only _Newsletter_ and _Testimonial_ patterns are displayed.

![Screenshot 2022-11-02 at 2 08 29 PM](https://user-images.githubusercontent.com/1190420/199593051-5165fc5d-f5b6-4d5c-b648-45357718c99c.jpg)
